controllers/storagecluster/cephcluster_test.go

```diff
@@ -332,39 +374,39 @@ func TestStorageClassDeviceSetCreation(t *testing.T) {
 		deviceSet := c.sc.Spec.StorageDeviceSets[0]
 
 		for i, scds := range actual {
-			assert.Equal(t, fmt.Sprintf("%s-%d", deviceSet.Name, i), scds.Name)
-			assert.Equal(t, deviceSet.Count/3, scds.Count)
-			assert.Equal(t, defaults.DaemonResources["osd"], scds.Resources)
-			assert.Equal(t, deviceSet.DataPVCTemplate, scds.VolumeClaimTemplates[0])
-			assert.Equal(t, true, scds.Portable)
-			assert.Equal(t, c.sc.Spec.Encryption.Enable, scds.Encrypted)
+			assert.Equal(t, fmt.Sprintf("%s-%d", deviceSet.Name, i), scds.Name, c.label)
```

Comment:
Don't do the addition of labels in this PR... this is something that needs to be discussed further so we can unify on a common strategy for reporting unit tests.